### ISSUE TITLE
CDX Import, CDXML Import: parsing error when superatom starts with 'R' symbol

### ISSUE THREAD

**Issue by Zhirnoff** [epam/Indigo#960]
**Steps to Reproduce**
1. Launch Ketcher
2. Paste from Clipboard or open [File](https://github.com/epam/Indigo/files/10336207/marvinjs_untitled_file.80.zip)

```
VmpDRDAxMDAEAwIBAAAAAAAAAAAAAAAAAAAAAAMAEAAAAENoZW1EcmF3IDcuMC4xCAAaAAAARXEgMS1QcmlucyBwcmVjZWRlbnQuY2R4AAMyAAgA////////AAAAAAAA//8AAAAA/////wAAAAD//wAAAAD/////AAAAAP////8AAP//DggCAPQBAQkIAAAAIwAAAFwAAgkIAAAAxgEAAEYCAggQAAAAJAAAACQAAAAkAAAAJAA6BAEAATsEAQAAPAQBAAAMBgEAAQ0GAQAABgcCAAAABwcCAAEAAwgEAAAAeAAFCAQAAGYOAAYIBAAAAAIABwgEAACaAAAICAQAAJoBAAkIBAAAgAIADAgBAAANCAAAIwgBAAUECAIAtAAKCAgAFQBgAMgAAwALCAgAFQAAAMgAAwAACHgAAAMAAALQAtAAAAAAHIQVkP7K/soduhayA2cFKAP8AAIAAALQAtAAAAAAHIAVkAABAAAAZAAAAAEAAQEBAAAAAScPAAEAAQAAAAAAAAAAAAAAAAACABkBkAAAAAAAQAAAAAAAAAAAAAEAAAAAAAAAAAAAAAAAAAAABAIQAACAYQAAAFgAELEuAQCAxAEAAR8AAQACABUAECcJAEhlbHZldGljYRcAECcGAFN5bWJvbAGAZBcAAAQCEAAAAAAAAAAAAAAA0AIAABwCDwgCAAEAEAgCAAEAFggEAAAAJAAYCAQAAAAkAAKAVxcAAAQCEADjKLYACqmfAK/U+QDcuQwBA4DaDgAABAIQAOMotgAKqZ8Ar9T5ANy5DAEEgN4OAAAKAAIAaQEAAggASXXIALVBxAA3BAEAAQAABIDfDgAACgACAGoBAAIIAEkPugC1QcQABoAAAAAAAAcNAAEAAAAVAGAAyAADAE8AAggAsPW9AAynwQAEAhAA4yi2AAynwQDjKL4ADGfJACMIAQAAAAA3BAEAAQIEAgAIACsEAgAAAAAABYAYDwAACgACAG4BAAYCAAIAAwYCAAIABAYEAN4OAAAFBgQA3w4AAAoGAQAAAAAEgOkOAAAKAAIAfgEAAggAFqLPACSv0AA3BAEAAAAABYAjDwAACgACAH8BBAYEAN4OAAAFBgQA6Q4AAAoGAQABAAAEgOoOAAAKAAIAgAEAAggAFgjeACSv0AA3BAEAAAAABIDrDgAACgACAIEBAAIIABY75QBQJ90ANwQBAAAAAASA7A4AAAoAAgCCAQACCAAWCN4Aep/pADcEAQAAAAAEgO0OAAAKAAIAgwEAAggAFqLPAHqf6QA3BAEAAAAABIDuDgAACgACAIQBAAIIABZvyABQJ90ANwQBAAEAAAWAJA8AAAoAAgCFAQAGAgACAAMGAgABAAQGBADpDgAABQYEAOoOAAAKBgEAAAsGEAApDwAAIw8AADYPAAAlDwAAAAAFgCUPAAAKAAIAhgEEBgQA6g4AAAUGBADrDgAACgYBAAEAAAWAJg8AAAoAAgCHAQAGAgACAAMGAgABAAQGBADrDgAABQYEAOwOAAAKBgEAAAsGEAAlDwAANw8AADkPAAAnDwAAAAAFgCcPAAAKAAIAiAEEBgQA7A4AAAUGBADtDgAACgYBAAEAAAWAKA8AAAoAAgCJAQAGAgACAAMGAgABAAQGBADtDgAABQYEAO4OAAAKBgEAAAsGEAAnDwAASQ8AAAAAAAApDwAAAAAFgCkPAAAKAAIAigEEBgQA7g4AAAUGBADpDgAACgYBAAEAAASA+Q4AAAoAAgCNAQACCADiNOUAtUHEAAaAAAAAAAAHEgABAAAAFQBgAMgAAwBPTzJTUGgBBwEA/wUHAQADAAIIAEkb6QA1bccABAIQAHxO4QD3oqAAfE7rADVtxwAjCAEA/wAAAAQCAAUANwQBAAEDgGUXAAAEAhAASOHHAGarvgCL2eoAJ90bAQSAqxAAAAoAAgAJAQACCAAVCN4AJK/QAAaAAAAAAAAHDQABAAAAFQBgAJYAAwBPAAIIAEj74ABXws0ABAIQAOIU2wBXws0AryHhACSP0wAjCAEAAAAANwQBAAACBAIACAArBAIAAAAAAASArBAAAAoAAgAKAQACCAC/MuUA4BjdAAaAAAAAAAAHDQABAAAAFQBgAJYAAwBPAAIIAPIl6AATLNoABAIQAIw/4gATLNoAWUzoAOD43wAjCAEAAAAANwQBAAACBAIACAArBAIAAAAAAASArRAAAAoAAgALAQACCAAVCN4AnYLpAAaAAAAAAAAHDQABAAAAFQBgAJYAAwBPAAIIAEj74ADQleYABAIQAOIU2wDQleYAryHhAJ1i7AAjCAEAAAAANwQBAAACBAIACAArBAIAAAAAAASArhAAAAoAAgAMAQACCAC/MuUAWez1AAaAAAAAAAAHDQABAAAAFQBgAJYAAwBTAAIIAPIl6ADyhfMABAIQAIw/4gDyhfMAWUzoAPKF+AAjCAEAAAAANwQBAAACBAIAEAArBAIAAAAAAASArxAAAAoAAgANAQACCAAVCN4AFVYCATcEAQAAAAAEgLAQAAAKAAIADgEAAggAvzLlANK/DgE3BAEAAAAABICxEAAACgACAA8BAAIIABUI3gCOKRsBNwQBAAAAAASAshAAAAoAAgAQAQACCAC/ss8AjikbATcEAQAAAAAEgLMQAAAKAAIAEQEAAggAFYjIANK/DgE3BAEAAAAABIC0EAAACgACABIBAAIIAL+yzwAVVgIBNwQBAAAAAASAvRAAAAoAAgATAQACCAC+MuUAZkXEAAAEAgAMADcEAQAAAAAFgL4QAAAKAAIAFAEEBgQAvRAAAAUGBACrEAAACgYBAAAAAAWAvxAAAAoAAgAVAQQGBACrEAAABQYEAKwQAAAKBgEAAAAABYDAEAAACgACABYBBAYEAKwQAAAFBgQArRAAAAoGAQAAAAAFgMEQAAAKAAIAFwEEBgQArRAAAAUGBACuEAAACgYBAAAAAAWAtRAAAAoAAgAYAQQGBACuEAAABQYEAK8QAAAKBgEAAAAABYC2EAAACgACABkBAAYCAAIAAwYCAAEABAYEAK8QAAAFBgQAsBAAAAoGAQAACwYQALsQAAC1EAAAAAAAALcQAAAAAAWAtxAAAAoAAgAaAQQGBACwEAAABQYEALEQAAAKBgEAAAAABYC4EAAACgACABsBAAYCAAIAAwYCAAEABAYEALEQAAAFBgQAshAAAAoGAQAACwYQALcQAAAAAAAAAAAAALkQAAAAAAWAuRAAAAoAAgAcAQQGBACyEAAABQYEALMQAAAKBgEAAAAABYC6EAAACgACAB0BAAYCAAIAAwYCAAEABAYEALMQAAAFBgQAtBAAAAoGAQAACwYQALkQAAAAAAAAAAAAALsQAAAAAAWAuxAAAAoAAgAeAQQGBACvEAAABQYEALQQAAAKBgEAAAAAAAAAAAWANg8AAAoAAgCOAQQGBADqDgAABQYEAPkOAAAKBgEAAQAABIARDwAACgACAI8BAAIIAK+U8wBQJ90ABoAAAAAAAAcQAAEAAAAVAGAAyAADAE9DSDMFBwEAAQACCAAWe/cA6UDZAAQCEABJru8A6UDZAK/U+QD01/MAIwgBAAAAAAAEAgAFADcEAQABA4BmFwAABIBnFwAAAAIIAB7G6ACp6tYAAgQCAAgAKwQCAAAAAAAEgGgXAAAAAggA6vLvABhY4wArBAIAAwAAAASAaxcAAAAEAgAMAAAABYBpFwAABAYEAGsXAAAFBgQAZxcAAAAABYBqFwAABAYEAGcXAAAFBgQAaBcAAAAAAAAAAAWANw8AAAoAAgCQAQQGBADrDgAABQYEABEPAAAKBgEAAQAABIATDwAACgACAJMBAAIIAOI05QDpDPYANwQBAAEAAAWAOQ8AAAoAAgCUAQQGBADsDgAABQYEABMPAAAKBgEAAQAABIBHDwAACgACAJwBAAIIAGt3yAA4CfYABoAAAAAAAAcQAAEAAAAVAGAAyAADAE9DSDMFBwEAAQACCADSXcwA0SLyAAQCEAAFkcQA0SLyAGu3zgDcuQwBIwgBAAAAAAAEAgAFADcEAQABA4BsFwAABIBtFwAAAAIIALdvyADHB/YAAgQCAAgAKwQCAAAAAAAEgG4XAAAAAggAhJzPADV1AgErBAIAAwAAAASAcRcAAAAEAgAMAAAABYBvFwAABAYEAHEXAAAFBgQAbRcAAAAABYBwFwAABAYEAG0XAAAFBgQAbhcAAAAAAAAAAAWASQ8AAAoAAgCdAQQGBADtDgAABQYEAEcPAAAKBgEAAQAABoBbDwAACgACAKMBAAcNAAEAAAAVAAAAyAADACsCBwIAAQAAAggAL8XaAAqpnwAEAhAAY/jSAAqpnwAvBd0A13WlAAAAAAAGgFIXAAAKAAIAqwEABw0AAQAAABUAAACMAAMANQIHAgABAAACCAD/TtAAzCK/AAQCEAD/zsoAzCK/AHem0QBl/MIAEAAmAAAAQ2hlbURyYXcgY2Fu1XQgaW50ZXJwcmV0IHRoaXMgbGFiZWwuAAAAAAKAYRcAAAQCEADJq6oAMbRhAMlRBQHqe6QAA4BPDwAABAIQAMmrqgAxtGEAyVEFAep7pAAEgNsOAAAKAAIAZgEAAggAY7jgAG1IbwA3BAEAADAEAQAHMQQQADQPAAAVDwAAAAAAABoPAAAAAASA3A4AAAoAAgBnAQACCABjHu8AbUhvADcEAQABAAAEgN0OAAAKAAIAaAEAAggAY1H2AJnAewA3BAEAAQAABIDgDgAACgACAGsBAAIIAGOF2QCZwHsANwQBAAAwBAEABzEEEABSDwAAGg8AABsPAAAAAAAAAAAFgBUPAAAKAAIAbAEEBgQA2w4AAAUGBADcDgAACgYBAAEAAAWAFg8AAAoAAgBtAQAGAgACAAMGAgABAAQGBADcDgAABQYEAN0OAAAKBgEAAAsGEAAVDwAAAAAAADgPAAAAAAAAAAAFgBoPAAAKAAIAbwEEBgQA4A4AAAUGBADbDgAACgYBAAEAAASA4Q4AAAoAAgBwAQACCADKK8sAmcB7ADcEAQABAAAFgBsPAAAKAAIAcQEEBgQA4A4AAAUGBADhDgAACgYBAAEAAASA4g4AAAoAAgByAQACCAD8/sMAKlNvADcEAQABAAAFgBwPAAAKAAIAcwEEBgQA4Q4AAAUGBADiDgAACgYBAAEAAASA4w4AAAoAAgB0AQACCAAwsrUAKlNvADcEAQAAMAQBAAcxBBAAHg8AAB0PAAAfDwAAAAAAAAAABYAdDwAACgACAHUBBAYEAOIOAAAFBgQA4w4AAAoGAQABAAAEgOQOAAAKAAIAdgEAAggAyouuANLwYgA3BAEAAQAABYAeDwAACgACAHcBAQYCAAIABAYEAOMOAAAFBgQA5A4AAAoGAQABAAAEgOUOAAAKAAIAeAEAAggAyouuAIK1ewA3BAEAAQAABYAfDwAACgACAHkBBAYEAOMOAAAFBgQA5Q4AAAoGAQABAAAEgOYOAAAKAAIAegEAAggAMLK1ANwXiAA3BAEAAQAABYAgDwAACgACAHsBBAYEAOUOAAAFBgQA5g4AAAoGAQABAAAEgOcOAAAKAAIAfAEAAggAMJKuAB5vlAAGgAAAAAAABw8AAQAAABUAYADIAAMAT0JuBQcBAAEAAggAlniyALeIkAAEAhAAyauqALeIkADJq7IA6nukACMIAQAAAAAABAIABQA3BAEAAQOAchcAAAQCEADqp5gABFJUAPzItwCdH5sABIDDEAAACgACAB8BAAIIADAisgB9I4cABoAAAAAAAAcNAAEAAAAVAGAAlgADAE8AAggAYxW1ALA2hAAEAhAA/S6vALA2hADJO7UAfQOKACMIAQAAAAA3BAEAAAIEAgAIACsEAgAAAAAABIDEEAAACgACACABAAIIAHO4pQB8o3EANwQBAAAAAASAxRAAAAoAAgAhAQACCAC3TpkA0XhqADcEAQAAAAAEgMYQAAAKAAIAIgEAAggAt06ZAHsjXAA3BAEAAAAABIDHEAAACgACACMBAAIIAHO4pQDR+FQANwQBAAAAAASAyBAAAAoAAgAkAQACCAAwIrIAeyNcADcEAQAAAAAEgMkQAAAKAAIAJQEAAggAMCKyANJ4agA3BAEAAAAABIDKEAAACgACACYBAAIIAHO4pQDR+H8ANwQBAAAAAASA0BAAAAoAAgAnAQACCAAvIrIA0XiVAAAEAgAMADcEAQAAAAAFgNEQAAAKAAIAKAEEBgQA0BAAAAUGBADDEAAACgYBAAAAAAWA0hAAAAoAAgApAQQGBADDEAAABQYEAMoQAAAKBgEAAAAABYDTEAAACgACACoBAAYCAAIAAwYCAAEABAYEAMQQAAAFBgQAxRAAAAoGAQAACwYQAM0QAADOEAAAAAAAANQQAAAAAAWA1BAAAAoAAgArAQQGBADFEAAABQYEAMYQAAAKBgEAAAAABYDVEAAACgACACwBAAYCAAIAAwYCAAEABAYEAMYQAAAFBgQAxxAAAAoGAQAACwYQANQQAAAAAAAAAAAAAMsQAAAAAAWAyxAAAAoAAgAtAQQGBADHEAAABQYEAMgQAAAKBgEAAAAABYDMEAAACgACAC4BAAYCAAIAAwYCAAEABAYEAMgQAAAFBgQAyRAAAAoGAQAACwYQAMsQAAAAAAAAAAAAAM0QAAAAAAWAzRAAAAoAAgAvAQQGBADEEAAABQYEAMkQAAAKBgEAAAAABYDOEAAACgACADABBAYEAMQQAAAFBgQAyhAAAAoGAQAAAAAAAAAABYAhDwAACgACAH0BBAYEAOYOAAAFBgQA5w4AAAoGAQABAAAEgPcOAAAKAAIAiwEAAggAlovZAP7aYgA3BAEAAQAABYA0DwAACgACAIwBAQYCAAUABAYEANsOAAAFBgQA9w4AAAoGAQABAAAEgBIPAAAKAAIAkQEAAggA/KoEAZnAewA3BAEAAQAABYA4DwAACgACAJIBBAYEAN0OAAAFBgQAEg8AAAoGAQABAAAEgFAPAAAKAAIAoAEAAggADrDgAFUqiAAGgAAAAAAABw4AAQAAABUAYADIAAMAT0gFBwEAAQACCAB0luQA7kOEAAQCEACnydwA7kOEAKfJ5AAhN5MAIwgBAAAAADcEAQABAgQCAAgAKwQCAAEAAAAFgFIPAAAKAAIAoQEBBgIABQAEBgQA4A4AAAUGBABQDwAACgYBAAEAAAAABoBQFwAACgACAKkBAAcNAAEAAAAVAAAAjAADADkCBwIAAQAAAggAb03YAI1jdQAEAhAAb83SAI1jdQAIh9kAjWN5ABAAJgAAAENoZW1EcmF3IGNhbtV0IGludGVycHJldCB0aGlzIGxhYmVsLgAAAAACgGIXAAAEAhAAL2W/APzmBwFjmPAA/E5BAQKAYhAAAAQCEAAvZb8A2XgNAWOY8ACmRTcBBoBYDwAACgACAKIBAAduAAgAAAAVAAAAyAADAAIAFQBgAMgAAwADABcAYADIAAMABAAVAGAAyAADAA4AFQACAMgAAwAPABUAAADIAAMAGgAVAEAAyAADABsAFQAAAMgAAwBCRjMgT0V0Mg1IT0FjDW4taGV4YW5lDSAwIG9DAQcBAAECBwIAAQAEBwoABAAJAA4AFwAcAAACCAD8MccApkUiAQQCEAAvZb8A2XgNAWOY8ACmRTcBEAAmAAAAQ2hlbURyYXcgY2Fu1XQgaW50ZXJwcmV0IHRoaXMgbGFiZWwuDAgBAAEAAAeAYBAAAAoAAgCoAQAKAgAHAAcKAgABAA4ABgBYDwAAIgQEAhAAOmzDAPyaHwE67MoA/JofAQAAAAAHgF8PAAAKAAIApQEACgIAAQABCgEAAAIKAQACIAoCAOgDBAIQAMn+1wD8mj8Byf7XAPyaCQEAAAAAAoBjFwAABAIQAMkVnQAcfDwByecSAd/StQEDgJAOAAAEAhAAyRWdABx8PAHJ5xIB39K1AQSAkQ4AAAoAAgAxAQACCABiItMAZGpUATcEAQAAMAQBAAcxBBAA0w4AALQOAAAAAAAAuQ4AAAAABICSDgAACgACADIBAAIIAGKI4QBkalQBNwQBAAAwBAEABzEEEADUDgAAtA4AALUOAAAAAAAAAAAEgJMOAAAKAAIAMwEAAggAYrvoAJDiYAE3BAEAADAEAQAHMQQQANcOAAC1DgAAAAAAALYOAAAAAASAlA4AAAoAAgA0AQACCABiiOEAu1ptATcEAQAAMAQBAAcxBBAAwg4AALYOAAC3DgAAAAAAAAAABICVDgAACgACADUBAAIIAGIi0wC7Wm0BBoAAAAAAAAcNAAEAAAAVAGAAyAADAE8FBwEAAQACCADJCNcAVHRpAQQCEAD8O88AVHRpAfw71wBUNHEBIwgBAAAAADcEAQABAgQCAAgAKwQCAAAAAAAEgJYOAAAKAAIANgEAAggAYu/LAJDiYAE3BAEAADAEAQAHMQQQALoOAAC4DgAAuQ4AAAAAAAAAAAWAtA4AAAoAAgA3AQQGBACRDgAABQYEAJIOAAAKBgEAAQAABYC1DgAACgACADgBBAYEAJIOAAAFBgQAkw4AAAoGAQABAAAFgLYOAAAKAAIAOQEEBgQAkw4AAAUGBACUDgAACgYBAAEAAAWAtw4AAAoAAgA6AQQGBACUDgAABQYEAJUOAAAKBgEAAQAABYC4DgAACgACADsBBAYEAJUOAAAFBgQAlg4AAAoGAQABAAAFgLkOAAAKAAIAPAEEBgQAlg4AAAUGBACRDgAACgYBAAEAAASAlw4AAAoAAgA9AQACCADJlb0AkOJgATcEAQABAAAFgLoOAAAKAAIAPgEBBgIABQAEBgQAlg4AAAUGBACXDgAACgYBAAEAAASAmA4AAAoAAgA/AQACCAD7aLYAInVUATcEAQABAAAFgLsOAAAKAAIAQAEEBgQAlw4AAAUGBACYDgAACgYBAAEAAASAmQ4AAAoAAgBBAQACCAAvHKgAInVUATcEAQAAMAQBAAcxBBAAvQ4AALwOAAC+DgAAAAAAAAAABYC8DgAACgACAEIBBAYEAJgOAAAFBgQAmQ4AAAoGAQABAAAEgJoOAAAKAAIAQwEAAggAyfWgAMoSSAE3BAEAAQAABYC9DgAACgACAEQBAQYCAAIABAYEAJkOAAAFBgQAmg4AAAoGAQABAAAEgJsOAAAKAAIARQEAAggAyfWgAHrXYAE3BAEAAQAABYC+DgAACgACAEYBBAYEAJkOAAAFBgQAmw4AAAoGAQABAAAEgJwOAAAKAAIARwEAAggALxyoANM5bQE3BAEAAQAABYC/DgAACgACAEgBBAYEAJsOAAAFBgQAnA4AAAoGAQABAAAEgJ0OAAAKAAIASQEAAggAL/ygABaReQEGgAAAAAAABw8AAQAAABUAYADIAAMAT0JuBQcBAAEAAggAluKkAK+qdQEEAhAAyRWdAK+qdQHJFaUA4p2JASMIAQAAAAAABAIABQA3BAEAAQOAcxcAAAQCEADqEYsA/XM5AfsyqgCUQYABBIB4EAAACgACANwAAAIIAC+MpAB0RWwBBoAAAAAAAAcNAAEAAAAVAGAAlgADAE8AAggAYn+nAKdYaQEEAhAA/JihAKdYaQHJpacAdCVvASMIAQAAAAA3BAEAAAIEAgAIACsEAgAAAAAABIB5EAAACgACAN0AAAIIAHMimAB0xVYBNwQBAAAAAASAehAAAAoAAgDeAAACCAC3uIsAyppPATcEAQAAAAAEgHsQAAAKAAIA3wAAAggAt7iLAHRFQQE3BAEAAAAABIB8EAAACgACAOAAAAIIAHMimADKGjoBNwQBAAAAAASAfRAAAAoAAgDhAAACCAAvjKQAdEVBATcEAQAAAAAEgH4QAAAKAAIA4gAAAggAL4ykAMqaTwE3BAEAAAAABIB/EAAACgACAOMAAAIIAHMimADKGmUBNwQBAAAAAASAiRAAAAoAAgDkAAACCAAujKQAyJp6AQAEAgAMADcEAQAAAAAFgIoQAAAKAAIA5QAEBgQAiRAAAAUGBAB4EAAACgYBAAAAAAWAgBAAAAoAAgDmAAQGBAB4EAAABQYEAH8QAAAKBgEAAAAABYCBEAAACgACAOcAAAYCAAIAAwYCAAEABAYEAHkQAAAFBgQAehAAAAoGAQAACwYQAIYQAACHEAAAAAAAAIIQAAAAAAWAghAAAAoAAgDoAAQGBAB6EAAABQYEAHsQAAAKBgEAAAAABYCDEAAACgACAOkAAAYCAAIAAwYCAAEABAYEAHsQAAAFBgQAfBAAAAoGAQAACwYQAIIQAAAAAAAAAAAAAIQQAAAAAAWAhBAAAAoAAgDqAAQGBAB8EAAABQYEAH0QAAAKBgEAAAAABYCFEAAACgACAOsAAAYCAAIAAwYCAAEABAYEAH0QAAAFBgQAfhAAAAoGAQAACwYQAIQQAAAAAAAAAAAAAIYQAAAAAAWAhhAAAAoAAgDsAAQGBAB5EAAABQYEAH4QAAAKBgEAAAAABYCHEAAACgACAO0ABAYEAHkQAAAFBgQAfxAAAAoGAQAAAAAAAAAABYDADgAACgACAEoBBAYEAJwOAAAFBgQAnQ4AAAoGAQABAAAEgJ8OAAAKAAIASwEAAggAL7XoACrIeQE3BAEAADAEAQAHMQQQAMIOAADDDgAAAAAAAMgOAAAAAAWAwg4AAAoAAgBMAQEGAgAFAAQGBACUDgAABQYEAJ8OAAAKBgEAAQAABICgDgAACgACAE0BAAIIAC8b9wAqyHkBNwQBAAAAAASAoQ4AAAoAAgBOAQACCAAvTv4AVkCGATcEAQAAAAAEgKIOAAAKAAIATwEAAggALxv3AIC4kgE3BAEAAAAABICjDgAACgACAFABAAIIAC+16ACAuJIBNwQBAAAAAASApA4AAAoAAgBRAQACCAAvguEAVkCGATcEAQABAAAFgMMOAAAKAAIAUgEABgIAAgADBgIAAQAEBgQAnw4AAAUGBACgDgAACgYBAAALBhAAyA4AAMIOAADVDgAAxA4AAAAABYDEDgAACgACAFMBBAYEAKAOAAAFBgQAoQ4AAAoGAQABAAAFgMUOAAAKAAIAVAEABgIAAgADBgIAAQAEBgQAoQ4AAAUGBACiDgAACgYBAAALBhAAxA4AANYOAADYDgAAxg4AAAAABYDGDgAACgACAFUBBAYEAKIOAAAFBgQAow4AAAoGAQABAAAFgMcOAAAKAAIAVgEABgIAAgADBgIAAQAEBgQAow4AAAUGBACkDgAACgYBAAALBhAAxg4AAEMPAAAAAAAAyA4AAAAABYDIDgAACgACAFcBBAYEAKQOAAAFBgQAnw4AAAoGAQABAAAEgK0OAAAKAAIAWAEAAggAlfXLAPb8RwE3BAEAAQAABYDTDgAACgACAFkBAQYCAAUABAYEAJEOAAAFBgQArQ4AAAoGAQABAAAEgK4OAAAKAAIAWgEAAggAL7XoAPb8RwEGgAAAAAAABw4AAQAAABUAYADIAAMAT1IBBwEA/wUHAQADAAIIAJab7ABPb0sBBAIQAMnO5AAcfDwByc7sAE9vSwEjCAEA/wAAAAQCAAUANwQBAAEDgHQXAAAEAhAAL5XeAOv8NQHYWe4AMBdXAQSAjBAAAAoAAgDuAAACCABiiOEAqABIAQaAAAAAAAAHDQABAAAAFQBgAJYAAwBPAAIIAJV75ADbE0UBBAIQAC+V3gDbE0UB/KHkAKjgSgEjCAEAAAAANwQBAAACBAIACAArBAIAAAAAAASAjRAAAAoAAgDvAAACCAANs+gAZGpUAQaAAAAAAAAHDQABAAAAFQBgAJYAAwBSAAIIAECm6wAwl1EBBAIQANq/5QAwl1EBQKbrADAXVwEjCAEAAAAAAAQCAAcANwQBAAAAAASAjxAAAAoAAgDwAAACCAALs+gA65Y7AQAEAgAMADcEAQAAAAAFgJAQAAAKAAIA8QABBgIABQAEBgQAjxAAAAUGBACMEAAACgYBAAAAAAWAkRAAAAoAAgDyAAQGBACMEAAABQYEAI0QAAAKBgEAAAAAAAAAAAWA1A4AAAoAAgBbAQEGAgAFAAQGBACSDgAABQYEAK4OAAAKBgEAAQAABICvDgAACgACAFwBAAIIAPtH/gC7Wm0BBoAAAAAAAAcNAAEAAAAVAGAAyAADAE8AAggAYi4CAVR0aQEEAhAAlWH6AFR0aQGVYQIBVDRxASMIAQAAAAA3BAEAAQIEAgAIACsEAgAAAAAABYDVDgAACgACAF0BBAYEAKAOAAAFBgQArw4AAAoGAQABAAAEgLAOAAAKAAIAXgEAAggAyacMAVZAhgEGgAAAAAAABxAAAQAAABUAYADIAAMAT0NIMwUHAQABAAIIADCOEAHvWYIBBAIQAGPBCAHvWYIByecSAfnwnAEjCAEAAAAAAAQCAAUANwQBAAEDgHUXAAAEgHYXAAAAAggATtkBAckBgAECBAIACAArBAIAAAAAAASAdxcAAAACCAAaBgkBN2+MASsEAgADAAAABIB6FwAAAAQCAAwAAAAFgHgXAAAEBgQAehcAAAUGBAB2FwAAAAAFgHkXAAAEBgQAdhcAAAUGBAB3FwAAAAAAAAAABYDWDgAACgACAF8BBAYEAKEOAAAFBgQAsA4AAAoGAQABAAAEgLEOAAAKAAIAYAEAAggA+xT3AJDiYAE3BAEAAQAABYDXDgAACgACAGEBAQYCAAIABAYEAJMOAAAFBgQAsQ4AAAoGAQABAAAEgLIOAAAKAAIAYgEAAggA+0f+AO8lnwE3BAEAAQAABYDYDgAACgACAGMBBAYEAKIOAAAFBgQAsg4AAAoGAQABAAAEgEEPAAAKAAIAmAEAAggAhYrhADwinwEGgAAAAAAABxAAAQAAABUAYADIAAMAT0NIMwUHAQABAAIIAOxw5QDVO5sBBAIQAB+k3QDVO5sBhcrnAN/StQEjCAEAAAAAAAQCAAUANwQBAAEDgHsXAAAEgHwXAAAAAggA6ILhAOYenwECBAIACAArBAIAAAAAAASAfRcAAAACCAC0r+gAVYyrASsEAgADAAAABICAFwAAAAQCAAwAAAAFgH4XAAAEBgQAgBcAAAUGBAB8FwAAAAAFgH8XAAAEBgQAfBcAAAUGBAB9FwAAAAAAAAAABYBDDwAACgACAJkBBAYEAKMOAAAFBgQAQQ8AAAoGAQABAAAEgFoXAAAKAAIArQEAAggA+60MAbtabQEGgAAAAAAABxEAAQAAABUAYADIAAMATzJTUGgBBwEAAQACCABilBABu1ptAQQCEACVxwgBnNVdAZXHEgHZ33wBIwgBAAEAAAEEAQACAAQCAAUANwQBAAEDgIEXAAAEgIIXAAAAAggAAeMBAZaoWgECBAIACAArBAIAAAAAAASAgxcAAAACCADODwkBJztOAQIEAgAIACsEAgAAAAAABICEFwAAAAIIAM4PCQEEFmcBAgQCABAAKwQCAAAAAAAEgIUXAAAAAggAAeMBAbnNQQErBAIAAAAAAASAhhcAAAACCABoifMAuc1BASsEAgABAAAABICHFwAAAAIIAJtc7ABLYDUBKwQCAAEAAAAEgIgXAAAAAggAaInzANzyKAErBAIAAQAAAASAiRcAAAACCAAC4wEB3PIoASsEAgABAAAABICKFwAAAAIIAM8PCQFLYDUBKwQCAAEAAAAEgJUXAAAABAIADAAAAAWAixcAAAQGBACVFwAABQYEAIQXAAAAAAWAjBcAAAQGBACCFwAABQYEAIMXAAAAAAWAjRcAAAQGBACCFwAABQYEAIQXAAAAAAWAjhcAAAQGBACDFwAABQYEAIUXAAAAAAWAjxcAAAQGBACFFwAABQYEAIYXAAAABgIAAgADBgIAAQAAAAWAkBcAAAQGBACGFwAABQYEAIcXAAAAAAWAkRcAAAQGBACHFwAABQYEAIgXAAAABgIAAgADBgIAAQAAAAWAkhcAAAQGBACIFwAABQYEAIkXAAAAAAWAkxcAAAQGBACJFwAABQYEAIoXAAAABgIAAgADBgIAAQAAAAWAlBcAAAQGBACFFwAABQYEAIoXAAAAAAAAAAAFgFsXAAAKAAIArgEEBgQArw4AAAUGBABaFwAACgYBAAEAAAAABoBRFwAACgACAKoBAAcNAAEAAAAVAAAAjAADADkCBwIAAQAAAggAuarKAKd1WQEEAhAAuSrFAKd1WQFT5MsAp3VdARAAJgAAAENoZW1EcmF3IGNhbtV0IGludGVycHJldCB0aGlzIGxhYmVsLgAABoBTFwAACgACAKwBAAcNAAEAAAAVAAAAjAADADUCBwIAAQAAAggAuarqAFKgawEEAhAAuSrlAFKgawExAuwA63lvARAAJgAAAENoZW1EcmF3IGNhbtV0IGludGVycHJldCB0aGlzIGxhYmVsLgAAAAAGgDwPAAAKAAIAlwEAB0wABAAAABUAAQDwAAMAAQAVAAAAyAADABEAFQABAPAAAwASABUAAADIAAMAOCwgUiA9IEFjLCAgIDc2JQ05LCBSID0gSCwgICAgIDEwJQEHAQACAgcCAAEABAcGAAIAEQAiAAACCADd3SABERJZAQQCEABDhBcBERJZARCxLgGqS6YBEAAmAAAAQ2hlbURyYXcgY2Fu1XQgaW50ZXJwcmV0IHRoaXMgbGFiZWwuDAgBAAIAAAaAXg8AAAoAAgCkAQAHDwABAAAAFQAAAPAAAwAoMSkCBwIAAQAAAggAluvTAAAAsQEEAhAA/JHKAAAAsQGWa9YAzIy/ARAAJgAAAENoZW1EcmF3IGNhbtV0IGludGVycHJldCB0aGlzIGxhYmVsLgAABoBhDwAACgACAKYBAAcNAAEAAAAVAAEA8AADADYCBwIAAQAAAggA3d0gAaoqgQAEAhAAQ4QXAaoqgQCpyiIBENGHABAAJgAAAENoZW1EcmF3IGNhbtV0IGludGVycHJldCB0aGlzIGxhYmVsLgAABoBiDwAACgACAKcBAAcNAAEAAAAVAAEA8AADADcCBwIAAQAAAggA3d0gAfbgvgAEAhAAQ4QXAfbgvgB29yIBXIfFABAAJgAAAENoZW1EcmF3IGNhbtV0IGludGVycHJldCB0aGlzIGxhYmVsLgAAB4BgFwAACgACAK8BAAoCAAEAAQoBAAIEAhAAAIBkAACAwQEAgGQAAABbAAAADoCWFwAAAQwMAGEXAABXFwAAYhAAAAIMBABjFwAABAwEAF8PAAAAAAAAAAA=
```

**Expected behavior**
The structure opens without errors.

**Actual behavior**
The structure is not opened if it has a R-Group. 

**Attachments**
![2023-01-03_14h45_19](https://user-images.githubusercontent.com/62464241/210351138-2461f2f7-11e8-4d93-b1ac-2338e8e68363.png)

**Indigo/Bingo version**  
[Indigo Toolkit](http://lifescience.opensource.epam.com/indigo/)
Version 1.9.0-rc.2.0-g40c4e3793-x86_64-linux-gnu-11.2.1

**Comment by Zhirnoff** [comment]
Bug is reproduced in Version 1.10.0-rc.2

https://user-images.githubusercontent.com/62464241/218068903-0953d8e1-8e43-447a-b135-3b08576984fc.mp4

**Comment by even1024** [comment]
Fixed in master and 1.10.rc.3. Please retest.
